tracer/test/snapshots/WcfTests.__metadataSchemaVersion=v0_binding=BasicHttpBinding_enableNewWcfInstrumentation=True_enableWcfObfuscation=False.verified.txt

```diff
@@ -143,15 +142,10 @@ at Samples.Wcf.Server.CalculatorService.EndServerAsyncAdd(IAsyncResult asyncResu
     Service: Samples.Wcf,
     Type: web,
     ParentId: Id_13,
-    Error: 1,
     Tags: {
       component: Wcf,
+      custom-tag: DispatchMessageInspector,
       env: integration_tests,
-      error.msg: Something happened,
-      error.stack:
-System.ServiceModel.FaultException: Something happened
-at Samples.Wcf.Server.CalculatorService.BeginServerAsyncAdd(Double n1, Double n2, Boolean throwsException, Boolean synchronouslyCompletes, AsyncCallback callback, Object state),
-      error.type: System.ServiceModel.FaultException,
```

Comment:
That doesn't look right? Shouldn't we still get those errors?

Reply:
You're right, this looks fishy and I missed this diff 😞 I'll take a look into this

Reply:
Fixed 😄 I changed the strategy to use the _previous_ instrumentation for adding exception data before it gets caught by an exception handler. However, the new integration files will start/stop the span outside of the IDispatchMessageInspector loop so they'll have access to the active span